src/index.scss

```diff
@@ -0,0 +1,15 @@
+@import './variables';
+@import '~bootstrap/scss/bootstrap';
```

Comment:
Let me suggest approach based on my experience with optimizing when using this lib:

1. at first instead importing `@import '~bootstrap/scss/bootstrap';` as is, you can copy that file and place it content in your `index.scss`. This allow your then comment all unused components, to decrease resulting file size.

2. second step is copy `variables`,  then comment everything, remove `!default` flag, and keep it as list of everything that you can override. But for me this approach is very optional and up to you. 

   Let me explain, early when I closely worked with big-sized projects based on bootstrap,  I created some kind of project settings like `project-variables` it was mostly colors and constant sizes (about ~30 constants), which then I used inside this copy-pasted `variables` to override. So it was like you have 1 level scope styling variables and second level adjusted by first level. By this it creates huge scalability. 

   2 examples: 

   - say, you want change `bootstrap` to `materialize`. Or 
   - you want update `gutter` size in one place, from which depends some related components. Of course it's more about responsiveness, then about pixel-perfect))

Reply:
I don't think we have to overly optimize the css output at such an early stage. 

The current setup allows to override bootstrap variables and use bootstrap as you want, without being bothered by copying and commenting the code from bootstrap library.

By copying bootstrap variables in our code we induce on ourselves the task of maintaining the file in sync with minor and major changes to a lib, and this is only one drawback of this optimisation.

I understand the reasoning behind your idea, but failing to see the advantages for our particular project case.

Reply:
ok, agree with this, sometimes it can creat additional works on bootstrap updates. but merging tools very simplify it. I'll mark as resolved. Let's consider it as sharing for future.